docs/components/_index.scss

```diff
@@ -1,4 +1,6 @@
 // load styles required to demonstrate each component
 
 @import "grid/index";
+@import "menu/spec";
```

Comment:
To keep things a bit more organized, please place "spec" files in a new section below the "index" files (don't mix "spec" and "index" in the same group of imports).

```scss
/// load styles required to demonstrate each component
@import "grid/index";
@import "typography/index";

/// load styles needed to test component visuals
@import "menu/spec";
```

Reply:
Changes are done.